hw/dv/data/common_sim_cfg.hjson

```diff
@@ -52,6 +52,7 @@
   build_opts: [// List multiple tops for the simulation
                "{sim_tops}",
                // Standard UVM defines
+               "+define+UVM",
```

Comment:
I feel it's good to define UVM, but where do you use or you will use it later?

Reply:
It's there in `hw/ip/prim/rtl/prim_assert.sv`. If you recall, we were previously using `UVM_PKG` macro lookup to use `uvm_error` instead of `$error`, but that didn't work with Xcelium since UVM was getting compiled after RTL. So we decided to add `UVM` instead. This is also done in the Makefile flow.

Reply:
I recalled that. Thanks for the explanation.